relayer/relays/beacon/header/syncer/json/beacon_json.go

```diff
@@ -0,0 +1,315 @@
+package json
```

Comment:
This package contains all the beacon structs necessary to generate the test data and output to the test data fixture JSON files, as well as the benchmark data file.